src/uwtools/drivers/ungrib.py

```diff
@@ -41,18 +42,27 @@ def __init__(
 
     # Workflow tasks
 
-    @task
-    def gribfile(self):
-        """
-        A symlink to the input GRIB file.
-        """
-        path = self._rundir / "GRIBFILE.AAA"
-        yield self._taskname(str(path))
-        yield asset(path, path.is_symlink)
-        infile = Path(self._driver_config["gfs_file"])
-        yield file(path=infile)
-        path.parent.mkdir(parents=True, exist_ok=True)
-        path.symlink_to(infile)
+    @tasks
+    def gribfiles(self):
+        """
+        Symlinks to all the GRIB files.
+        """
+        yield self._taskname("gribfiles")
+        gfs_files = self._driver_config["gfs_files"]
+        offset = abs(gfs_files["offset"])
+        endhour = gfs_files["forecast_length"] + offset + 1
+        interval = gfs_files["interval_hours"]
+        cycle_hour = int((self._cycle - timedelta(hours=offset)).strftime("%H"))
+        suffix = "AAA"
+        links = []
+        for boundary_hour in range(offset, endhour, interval):
+            infile = Path(
+                gfs_files["path"].format(cycle_hour=cycle_hour, forecast_hour=boundary_hour)
+            )
+            link_name = self._rundir / f"GRIBFILE.{suffix}"
+            links.append((infile, link_name))
+            suffix = incr_str(suffix)
+        yield [self._gribfile(infile, link) for infile, link in links]
 
     @task
     def namelist_file(self):
```

Comment:
Do `end_date` and `interval_seconds` need to be set differently if we're processing multiple GRIB files? I'm looking at slides 13-20 of [this PDF](https://mmg.atm.ucdavis.edu/wp-content/uploads/2014/10/WPS-Duda.pdf) and am not sure. Or is this out-of-scope for this PR?